database/factories/LegiScan/BillFactory.php

```diff
@@ -2,16 +2,22 @@
 
 namespace Database\Factories\LegiScan;
 
+use App\Models\LegiScan\Bill;
 use Illuminate\Database\Eloquent\Factories\Factory;
 
 /**
  * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\LegiScan\Bill>
  */
 class BillFactory extends Factory
 {
+    public $model = Bill::class;
+
     public function definition()
     {
+        static $increment = 1;
+
         return [
+            'bill_id'              => $increment++,
```

Comment:
Using `$this->makeId()` fails because LegiScan doesn't use a column named `id`.

Maybe this macro could be updated to take a column name like `$this->makeId('bill_id')`.